doc/doxygen/src/MainPage.dox

```diff
@@ -1,5 +1,27 @@
 /*! @mainpage An Introduction
  *
+ * @note Looking for resources?
```

Comment:
[consider] I think I would re-order these a bit.

1. PyPI
2. Manager template
3. USD/OTIO/Ayon
4. Notebooks
5. Building
6. Codebase

I think the Notebooks may move up the list, but only when they offer more holistic coverage.